src/views/AddExpenses.js

```diff
@@ -1,10 +1,16 @@
 import { useState, useRef, useEffect } from 'react';
 import { convertEurosToCents, getUKFormattedDate } from '../utils/helpers';
+import DatePicker from 'react-datepicker';
+
+import 'react-datepicker/dist/react-datepicker.css';
```

Comment:
mmm, adding this stylesheet seems a little "burried" here. maybe we can put it somewhere so we do not have to remember adding this every time we use the date picker?

Reply:
sure I can check... libraries are usually quite picky about where the css import is placed... secondly do you have anything against installing `sass` into our project guys? the time will come when we will need to add some vanilla css or customise the bootstrap classes...

Reply:
i am not opposed to sass (i use it on a daily basis). however, I am typically very careful with introducing "new things". let's add it when we really need it and there is good justification to do so. (I am always worried about complexity and despite sass being relatively simple, it _is_ another technology on top of what we do have already)

Reply:
👍